packages/medusa/src/api/routes/admin/order-edits/request-confirmation.ts

```diff
@@ -65,15 +68,24 @@ export default async (req, res) => {
   await manager.transaction(async (transactionManager) => {
     await orderEditService
       .withTransaction(transactionManager)
-      .requestConfirmation(id, { loggedInUserId: loggedInUser })
+      .requestConfirmation(id, validatedBody.description, {
```

Comment:
suggestion: Replace with `payment_collection_description` or `payment_description` to eliminate ambiguity. 

Also, would it make sense to include the description in `context` or pass an object (with `description`) to make the method less resilient to future changes? E.g. in case we add more props.